internal/provider/data_source_pullzone_test.go

```diff
@@ -0,0 +1,36 @@
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourcePullZone(t *testing.T) {
+	resourceName := randResourceName()
+	resource.UnitTest(t, resource.TestCase{
+		Providers: testProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourcePullZone(resourceName),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
Isn't `resource.ComposeTestCheckFunc` unnecessary here? Only one TestCheck function is passed